### DISCUSSION TITLE
ILSpy support for .NET 8

### DISCUSSION THREAD

**Discussion by greenozon** [icsharpcode/ILSpy#2942] [Development]
Are there any plans to start supporting [.NET 8](https://learn.microsoft.com/en-us/dotnet/core/whats-new/dotnet-8)?

It's gonna be LTS edition (for 3 years instead of 1), so worth some focus ...

Some packages are already available for [downloads](https://dotnet.microsoft.com/en-us/download/dotnet/8.0)

**Comment by christophwille** [comment]
Not really - C# 12 features maybe late in Q3, and switching the build target 6 to 8 post-DotNetConf 23 in November.
